modules/toolchains_riscv_gnu/0.0.2/MODULE.bazel

```diff
@@ -0,0 +1,32 @@
+"""ARM cross compilation toolchains for bazel"""
+
+module(
+    name = "toolchains_riscv_gnu",
+    version = "0.0.2",
+    compatibility_level = 1,
+)
+
+bazel_dep(name = "platforms", version = "0.0.8")
+bazel_dep(name = "rules_cc", version = "0.0.9")
+
+riscv_toolchain = use_extension("@toolchains_riscv_gnu//:extensions.bzl", "riscv_toolchain")
+riscv_toolchain.riscv_none_elf(version = "13.2.0-2")
+use_repo(
+    riscv_toolchain,
+    "riscv_none_elf",
+    "riscv_none_elf_darwin_arm64",
```

Comment:
I looked at the release notes and their Bzlmod setup instructions and saw that they also include `use_repo`ing these additional repos. Is that necessary for end users? That could make it more difficult to change anything about supported platforms in a backwards compatible way.

Reply:
No need to change anything, it was just a drive-by comment trying to reduce possibly unnecessary API surface.

Reply:
It was a very good comment, I'll think about a way to remove those additional repos. Thanks for the feedback

Reply:
I found a way to integrate your suggestion, but if that's ok I will release that separately since it's an API change